lighthouse-cli/test/lantern.smoke.js

```diff
@@ -0,0 +1,87 @@
+/**
+ * @license Copyright 2019 Google Inc. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+'use strict';
+
+const config = {
+  extends: 'lighthouse:full',
+  settings: {
+    onlyCategories: ['performanceormance'],
+    precomputedLanternData: {
+      additionalRttByOrigin: {
+        'http://localhost:10200': 500,
+      },
+      serverResponseTimeByOrigin: {
+        'http://localhost:10200': 1000,
+      },
+    },
+  },
+};
+
+module.exports = [
+  {
+    batch: 'performance',
+    url: 'http://localhost:10200/online-only.html',
```

Comment:
config, url, batch, and assertions are all colocated, no more jumping around to 5 files! 🎉